src/pages/preferences/preferencesTabs/cliTab/DownloadCard.tsx

```diff
@@ -130,61 +135,40 @@ const descriptions = {
   "OSX ARM 64-bit": "M1 CPU",
 };
 const prettyDisplayNameTop = {
-  "OSX 64-bit": "macOS",
   "OSX ARM 64-bit": "macOS ARM",
   "Windows 64-bit": "Windows",
   "Linux 64-bit": "Linux (64-bit)",
 };
 
-const prettyDisplayNameAccordion = {
-  "Linux 64-bit": "Linux (64-bit, Legacy)",
-};
-
 const filterBinaries = (binary: ClientBinary) =>
-  /darwin_arm64\/|darwin_amd64\/|linux_amd64\/|windows_amd64\//.test(
-    binary.url
-  );
-
-// @ts-expect-error
-const Container = styled(Card)`
-  padding: ${size.m};
-` as typeof Card;
+  /darwin_arm64\/|linux_amd64\/|windows_amd64\//.test(binary.url);
 
 const CardGroup = styled.div`
   display: flex;
-  align-items: space-between;
+  gap: ${size.xs};
   margin-bottom: ${size.s};
 `;
 
-// @ts-expect-error
-const CliDownloadCard = styled(Card)`
+const CliDownloadCard = styled<CardType>(Card)`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   padding: ${size.s};
-  margin-right: ${size.xs};
-` as typeof Card;
+`;
 
 // @ts-expect-error
 const CliDownloadButton = styled(Button)`
   align-self: flex-start;
+  margin-top: ${size.xs};
 `;
```

Comment:
I think we can delete this style if we add `align-items: flex-start; gap: ${size.xxs};` into `CliDownloadCard`. maybe that's going a little css crazy though 🙂

Reply:
I actually went with `margin-top` so that the gap would be between just the button and the subheading, not between the title, subheading, and button. just being nitpicky here tho 😂